src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -60,18 +59,19 @@
     (> new-value prev-value)))
 
 (defn- f-view-internal
-  ;; crypto-decimals and limit-crypto args are needed for component tests only
-  [{:keys [crypto-decimals limit-crypto]}]
+  [{:keys [crypto-decimals limit-crypto type]}]
   (let [bottom                    (safe-area/get-bottom)
+        navigate-back-route       (case type
+                                    :bridge :wallet-bridge-send
+                                    :wallet-send-input-amount)
```

Comment:
I think having these type of checks is not scalable, maybe we would end having lots of these ones all over the file when we start to add more functionality or if we need this screen to support another transaction type. Maybe a good idea would be to abstract this into a common component, and then define two different screens with different configurable parameters, similar to what we did with `status-im.common.scan-qr-code.view` namespace. What do you think?